applications/plugins/SofaPython/Binding_BaseObject.cpp

```diff
@@ -149,6 +153,7 @@ SP_CLASS_METHOD(BaseObject,getPathName)
 SP_CLASS_METHOD(BaseObject,getLinkPath)
 SP_CLASS_METHOD(BaseObject,getSlaves)
 SP_CLASS_METHOD(BaseObject,getName)
+SP_CLASS_METHOD(BaseObject,getAsACreateObjectParameter)
```

Comment:
Are you sure ? Base does not have getPathName... so I doubt you can factor this behavior at that level.
To me the function needs to be in BaseObject and in Node.